app/_landing_pages/gateway/deployment-topologies.yaml

```diff
@@ -8,12 +8,292 @@ rows:
   - header:
       type: h1
       text: "Deployment Topologies"
-  - columns:
+    columns:
       - blocks:
           - type: structured_text
             config:
               blocks:
                 - type: text
                   text: |
-                    @todo
-                    pull content from https://docs.konghq.com/gateway/latest/production/deployment-topologies/, cover what a deployment topology is along with links to more info about each topology
\ No newline at end of file
+                    Kong provides different modes for you to deploy {{site.konnect_short_name}} and {{site.base_gateway}} in. 
+                    Each mode has benefits and limitations, so it is important to consider them carefully when deciding which mode to use to in production installations.
+  - header:
+      type: h2
+      text: "Deployment topology support"
+    columns:
+      - blocks:
+          - type: feature_table
+            config:
+              columns:
+                - title: "{{site.konnect_short_name}}"
+                  key: konnect
+                - title: "{{site.base_gateway}}"
+                  key: gateway
+                - title: Kubernetes
+                  key: k8s
```

Comment:
What is Kubernetes in this context?

Reply:
@mheap It was meant to represent Kong on/with Kubernetes: https://docs.konghq.com/gateway/latest/production/deployment-topologies/kubernetes/